2-WebApp-graph-user/2-5-HybridFlow/README.md

```diff
@@ -0,0 +1,480 @@
+* A user account in your **Azure AD** tenant. This sample will not work with a **personal Microsoft account**.  If you're signed in to the [Azure portal](https://portal.azure.com) with a personal Microsoft account and have not created a user account in your directory before, you will need to create one before proceeding.
+
+## Setup the sample
+
+### Step 1: Clone or download this repository
+
+From your shell or command line:
+
+```console
+    git clone https://github.com/Azure-Samples/active-directory-aspnetcore-webapp-openidconnect-v2.git
+```
+
+or download and extract the repository .zip file.
+
+>:warning: To avoid path length limitations on Windows, we recommend cloning into a directory near the root of your drive.
+
+### Step 2: Navigate to project folder
+
+```console
+    cd 2-WebApp-graph-user\2-5-HybridFlow
+```
+
+### Step 3: Application Registration
+
+There are two projects in this sample. Each needs to be separately registered in your Azure AD tenant. To register these projects:
+
+You can follow the [manual steps](#Manual-steps)
+
+**OR**
+
+### Run automation scripts
+
+* use PowerShell scripts that:
+  * **automatically** creates the Azure AD applications and related objects (passwords, permissions, dependencies) for you.
+  * modify the projects' configuration files.
+
+  <details>
+   <summary>Expand this section if you want to use this automation:</summary>
+
+    > **WARNING**: If you have never used **Azure AD Powershell** before, we recommend you go through the [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md) once to ensure that your environment is prepared correctly for this step.
+  
+    1. On Windows, run PowerShell as **Administrator** and navigate to the root of the cloned directory
+    1. In PowerShell run:
+
+       ```PowerShell
+       Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope Process -Force
+       ```
+
+    1. Run the script to create your Azure AD application and configure the code of the sample application accordingly.
+    1. For interactive process - in PowerShell run:
+
+       ```PowerShell
+       cd .\AppCreationScripts\
+       .\Configure.ps1 -TenantId "[Optional] - your tenant id" -Environment "[Optional] - Azure environment, defaults to 'Global'"
+       ```
+
+       > Other ways of running the scripts are described in [App Creation Scripts guide](./AppCreationScripts/AppCreationScripts.md)
+       > The scripts also provide a guide to automated application registration, configuration and removal which can help in your CI/CD scenarios.
+
+  </details>
+
+### Manual Steps
+
+ > Note: skip this part if you've just used Automation steps
+
+Follow the steps below for manually register and configure your apps
+
+<details>
+   <summary>Expand this section if you want to use this automation:</summary>
```

Comment:
><summary>Expand this section if you want to use this automation:</summary> [](http://example.com/codeflow?start=3&length=74)

[code gen bug]<summary>Expand this section to manually register your app(s):</summary>